File: cfgs/pythoncfgs/EFFICIENT_SEARCH_IN_AN_ARRAY_WHERE_DIFFERENCE_BETWEEN_ADJACENT_IS_1.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;foo_1 <= foo_2 - 1;9;
3;statement;return -1;13;
4;conditional;foo_1[foo_2] == foo_3;10;
5;statement;foo_1 += fun_1(foo_2[foo_3] - foo_4);12;
6;statement;return foo_1;11;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;5;6;
4;2;
5;7;
6;

File: cfgs/java_cfgs_foo/EFFICIENT_SEARCH_IN_AN_ARRAY_WHERE_DIFFERENCE_BETWEEN_ADJACENT_IS_1.java
Nodes
1;special;;;
2;statement;int i = 0;14;
3;conditional;foo_1 <= (foo_2 - 1);15;
4;conditional;foo_1[foo_2] == foo_3;16;
5;statement;return foo_1;16;
6;exit;;;
7;statement;i += fun_1(foo_1[foo_2] - foo_3);17;
8;statement;return -1;19;
Edges
0;2;
1;3;
2;8;4;
3;5;7;
4;6;
5;
6;3;
7;6;
